media/stylus/includes/vars.styl

```diff
@@ -173,7 +173,7 @@ $selector-heading-font-fallback = 'html[data-ffo-opensanslight="false"]:not(.no-
 paths
 ====================================================================== */
 $path-to-assets = '/media/';
-$path-to-images = $path-to-assets + 'img/';
+$path-to-images = '/static/img/';
```

Comment:
This should be a relative path from the compiled CSS files to the image file so we can use of staticfiles' included replacement of full file names (`url('../img/header.png')` to `url('../img/header.ddaac5ce6246.png')`. The relativeness is needed to accurately calculate a map of files in all CSS files something if we'd hard code the root path would not work depending on the storage backend we use.